src/main/java/edu/ucdavis/dss/ipa/services/jpa/JpaBudgetScenarioService.java

```diff
@@ -107,6 +147,17 @@ public BudgetScenario update(BudgetScenario budgetScenario) {
         return budgetScenarioRepository.save(budgetScenario);
     }
 
+    @Override
+    public void createLineItemsFromTeachingAssignment(TeachingAssignment teachingAssignment) {
+        if (teachingAssignment.isApproved() && (teachingAssignment.isBuyout() || teachingAssignment.isWorkLifeBalance())) {
+
```

Comment:
Don't start an if block with a blank line. But a blank line before the 'for' on line 155 would work.